lib/logstash/patches/silence_concurrent_ruby_warning.rb

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "concurrent/concern/logging"
+require "concurrent/concern/deprecation"
+require "concurrent/version"
+require "cabin"
+
+# Concurrent-ruby is throwing warning when the code is run under jdk7, and they
+# will provide best effort support, logstash has to support JDK7 for a few months.
+#
+# By default all deprecation warnings of the concurrent ruby
+# library use the `WARN` level which is show everytime we boot logstash,
+# This monkeypatch change the log level of the deprecation warning to be `debug`
+# instead. This monkey patch might be a bit over kill but there is no
+# easy way to override the java version check.
+#
+# ref: https://github.com/ruby-concurrency/concurrent-ruby/blob/v0.9.1/lib/concurrent/configuration.rb#L284-L295
+#
+# This patch is only valid for 0.9.1
+if Concurrent::VERSION == "0.9.1" 
+  module Concurrent
+    module Concern
+      module Deprecation
+        include Concern::Logging
+
+        def deprecated(message, strip = 2)
```

Comment:
Well, I have been **bite** in the past by `concurrent-ruby` updates, I would prefer a way to at least debug if something changes. what do you think?